src/django_management/urls.py

```diff
@@ -15,8 +15,13 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.shortcuts import redirect
+from django.urls import include, path
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('accounts/', include('accounts.urls', namespace='accounts')),
+
+    # Redirect the root URL to the login page
+    path('', lambda request: redirect('login')),
```

Comment:
Since we added the app_name to `accounts`, can you modify this with redirect('accounts/login')